plaidml_build_onnx_node_tests.sh

```diff
@@ -17,23 +17,23 @@ START=$(date +%s)
 
 git clone --recursive --branch plaidml-v1 https://github.com/plaidml/plaidml.git ./build/plaidml
 cd build/plaidml/
-./configure
-conda activate .cenv/
-bazelisk build plaidml:shlib
-conda deactivate
-
 
 # Set environment variables so that onnxruntime can find plaidml 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; 
```

Comment:
What if we got rid of this condition? I think it might just work on mac fine (mac generates `.so`)

Reply:
So upon further investigation... the .so and the .dylib are both generated on Mac, except that there is no .dylib under Bazel-bin (like there used to be before the recent Bazel build rule change) and instead sits deep inside Bazel-out (I did not try to investigate why this is yet and if it can be/ should be remedied) 

I did try getting rid of the condition and building on Mac using the .so. This should probably work, but with the current setup of the onnxruntime build,  it does not. it throws a python subprocess error. (I did not investigate this since using the dylib works for now and this is only needed for my local builds when I lose access to vpn and need to keep working locally on my Mac)

Reply:
The `.so` is *not* generated on Mac. It's just that bazel doesn't cleanup old files.